### DISCUSSION TITLE
how do i create an organization

### DISCUSSION THREAD

**Discussion by baekteun** [community/community#18777] [Other Feature Feedback, Questions, & Ideas]
how do i create an organization

**Comment by Zi-Tao** [comment]
@baekteun https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch

**Comment by TenTraicion** [comment]
Go to this link: [Creating Organization for Free in Github](https://github.com/account/organizations/new?plan=free&ref_cta=Create%2520a%2520free%2520organization&ref_loc=cards&ref_page=%2Forganizations%2Fplan)

I have selected the free plan already for you there. All you have to do is Give the organization a name, put a valid email, type and verify yourself. Then accept the terms and submit. Voila!

![image](https://user-images.githubusercontent.com/103281314/176932541-c79b14be-6113-466e-8b67-b518515491e7.png)

If you want to create organization with another plan, just click on the "+" icon beside your profile and select new organization. then select your plan and go with the flow.

Hope it helps. Good Luck!
